lib/rgeo/cartesian/factory.rb

```diff
@@ -99,11 +109,10 @@ def marshal_dump # :nodoc:
       end
 
       def marshal_load(data) # :nodoc:
-        if (coord_sys_data = data["cs"])
-          coord_sys = CoordSys::CONFIG.default_coord_sys_class.create_from_wkt(coord_sys_data)
-        else
-          coord_sys = nil
-        end
+        coord_sys =
+          if (coord_sys_data = data["cs"])
+            CoordSys::CONFIG.default_coord_sys_class.create_from_wkt(coord_sys_data)
+          end
```

Comment:
This one seemed a bit hard to grasp to me, any opinion on that option?

```ruby
cs_class = CoordSys::CONFIG.default_coord_sys_class
coord_sys = data["cs"]&.then { |cs| cs_class.create_from_wkt(cs) }
```

Not so sure though

Reply:
I agree it wasn't that easy to understand on first glance, but I was just trying to stick to fixing the rubocop issues instead of updating the code.

I was thinking of

```ruby
coord_sys_data = data["cs"]
coord_sys =
  if coord_sys_data
    CoordSys::CONFIG.default_coord_sys_class.create_from_wkt(coord_sys_data)
  end
```

but I like the way you have it better.

Reply:
@thestelz both look good to me, and anyway this is non-blocking. Do as you feel 🙂